difflint/lint.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2015 Endless Mobile, Inc.
+
+import os.path
+from pkg_resources import resource_filename
+import shutil
+import subprocess
+
+
+def get_missing_linters():
+    '''
```

Comment:
nit: I believe `pep8` recommends keeping this on the same line as the comment but does want (as you have it) a newline between the end of the comment and the closing triple single quotes (multiple instances).